pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -2011,4 +2033,34 @@ public ConfigField(Field field) {
             return Optional.empty();
         }
     }
+
+    private void checkMessagePublishBuffer() {
+        currentMessagePublishBufferSize = 0;
```

Comment:
It seems to me that this variable doesn't have to be a class variable of `BrokerService`. It can just be a local variable, right?

Reply:
Yes, it is.